content-security-policy/support/testharness-helper.sub.js

```diff
@@ -0,0 +1,47 @@
+const Host = {
+  SAME_ORIGIN: "same-origin",
+  CROSS_ORIGIN: "cross-origin",
+};
+
+const PolicyHeader = {
+  CSP: "echo-policy.py?policy=",
+  EMBEDDING_CSP: "echo-embedding-csp.py",
+} 
+
+function generateURL(host, path) {
+  var url = new URL("http://{{host}}:{{ports[http][0]}}/content-security-policy/support/");
```

Comment:
you'll need either to remove the `.https` suffix from the test, or change this to load over HTTPS (by changing the protocol to `https`, and the port to `ports[https][0]`.